src/main/java/org/opentripplanner/model/plan/Leg.java

```diff
@@ -58,6 +58,13 @@
     */
    public Boolean realTime = false;
 
```

Comment:
Ask @hannesj Why FlexTrip does not extend Trip

Reply:
I had a discussion with Hannes about this, and he agreed that it would probably be a good idea to have FlexTrip extend Trip. It is not quite straightforward, so we can handle it in another PR.